frontend/src/pages/profile/components/user-activity/user-activity.tsx

```diff
@@ -13,18 +18,27 @@ type Properties = {
 
 const UserActivity: React.FC<Properties> = ({ userActivity }) => {
   const uniqueMonths: string[] = getUniqueMonths(userActivity);
+  const dateFormatter = new Intl.DateTimeFormat(DATE_LOCALE, DATE_OPTIONS);
 
-  const activityStatistic = userActivity.map((activity) => {
+  const activityStatistic = userActivity.map((activity, index) => {
     const { count, date } = activity;
     const activityBreakpoint = getActivityBreakpoint(count);
+    const localDate = dateFormatter.format(new Date(date));
+    const activityTitle = `${count} action${
+      count > 1 ? 's' : ''
+    } on ${localDate}`;
```

Comment:
I believe in `getPublicationTime` helper - there is already the same check to add pluralEnding -- let's move it to separate helper and use in both places